avalon/Gemfile.local

```diff
@@ -1 +1,3 @@
 gem 'pg'
+gem 'tzinfo-data'
```

Comment:
According to http://tzinfo.github.io/ tzinfo-data is only needed if a zoneinfo directory doesn't exist on the system.  I would vote for adding tzinfo-data to the upstream Gemfile so we don't assume that zoneinfo is present and hopefully work on in more environments.